hadoop-ozone/dist/src/main/smoketest/s3/objecttagging.robot

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation       S3 gateway test with aws cli
+Library             OperatingSystem
+Library             String
+Resource            ../commonlib.robot
+Resource            commonawslib.robot
+Test Timeout        5 minutes
+Suite Setup         Setup s3 tests
+
+*** Variables ***
+${ENDPOINT_URL}       http://s3g:9878
+${OZONE_TEST}         true
+${BUCKET}             generated
+
+
+*** Test Cases ***
+
+Put object tagging
```

Comment:
Would be nice to add negative test cases (e.g. request for non-existent key).  Can be done in follow-up.

Also would be nice to create reusable keywords to avoid duplication.  Also can be in follow-up.

Reply:
Thanks for the review. I added for case non-existent key. The other cases like empty tag key and empty tag values are usually thrown in the AWS SDK client-side. I added the validation server-side just in case.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-11678 for the extract the object tagging keywords.